app/decorators/stream_item_decorator.rb

```diff
@@ -1,33 +1,48 @@
 class StreamItemDecorator < Draper::Decorator
-
   MAX_BODY_SIZE = 250
   MAX_PICTURES = 10
 
-  # TODO i18n
-
   delegate_all
 
   def publishable?
-    shared? and not (streamable_type == 'Message' and group_id.nil?)
+    shared? && !(streamable_type == 'Message' && group_id.nil?)
   end
 
-  def to_html(options={})
+  def to_html(options = {})
```

Comment:
Unused method argument - `options`. If it's necessary, use `_` or `_options` as an argument name to indicate that it won't be used. You can also write as `to_html(*)` if you want the method to accept any arguments but don't care about them.<br>Assignment Branch Condition size for to_html is too high. [28.16/18]